dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/AbstractCADriver.java

```diff
@@ -44,6 +44,10 @@ protected void printBanner() {
 		for (String uri : uris) {
 			banner.append("*      " + uri + "\n");
 		}
+
+		banner.append(
+				"* ACME CA Server Ocsp Responder URI (for revocation checking): " + caContainer.getOcspResponderUrl()
```

Comment:
This will throw a UnsupportedOperationException when running Pebble.

Reply:
Resolved